examples/http-to-websocket-upgrade/http_to_websocket_upgrade.bal

```diff
@@ -31,7 +31,7 @@ service<http:Service> httpService bind servicEp {
             }
         }
 
-        caller -> respond(resp) but { error e => log:printError("Error sending message", err = e) };
+        caller -> respond(resp) but { error e => log:printError("Error in responding", err = e) };
     }
```

Comment:
shouldn't we remove the spaces around '->' and include a space after "responding" ?

```caller->respond(resp) but { error e => log:printError("Error in responding ", err = e) };```